Modules/_sha3/sha3module.c

```diff
@@ -487,48 +500,24 @@ static PyGetSetDef SHA3_getseters[] = {
     {NULL}  /* Sentinel */
 };
 
+#define SHA3_TYPE_SLOTS(type_slots_obj, type_doc, type_methods) \
+    static PyType_Slot type_slots_obj[] = { \
+        {Py_tp_dealloc, SHA3_dealloc}, \
```

Comment:
When you convert a static type to a heap type, you have to DECREF its type in its dealloc method. Example:

```
    (...)
    PyTypeObject *tp = Py_TYPE(self);
    tp->tp_free(self);
    Py_DECREF(tp);
```

Reply:
@vstinner Thanks for the reminder.  I have this in my notes but somehow missed it for this module.